modules/040-control-plane-manager/images/kubernetes-api-proxy/werf.inc.yaml

```diff
@@ -1,6 +1,11 @@
 ---
 image: {{ $.ModuleName }}/{{ $.ImageName }}
 from: {{ env "BASE_NGINX_ALPINE" }}
+import:
```

Comment:
I did not get this part. Why do we need the pause inside the Nginx image?

Reply:
I did it because i added kubernetes-api-proxy to image-holder. I think it is an important part of control-plane.

Reply:
We need to pull kubernetes-api-proxy image before imagePullSecret exists. So we are using the same scheme as for control-plane images.

Reply:
It can be excessive, so i removed pause binary from container and removed kubernetes-api-proxy from image-holder.